src/Nox.Types/Types/Color/Color.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents a Nox <see cref="Color"/> type and value object. 
+/// </summary>
+public class Color : ValueObject<byte[], Color>
+{
+    private byte _alpha { get; set; }
+    private byte _red { get; set; }
+    private byte _green { get; set; }
+    private byte _blue { get; set; }
+    
+    public Color()
+    {
+        Value = new byte[4];
+    }
+
+    private bool _isEmpty { get; set; }
```

Comment:
I'm not sure what is this property for since it's never updated.

Reply:
Its raised from converter, if the converter wasn't capable of create a new Color type.